src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-delegate.h

```diff
@@ -50,6 +50,8 @@ class Delegate
      * @return CHIP_ERROR_PROVIDER_LIST_EXHAUSTED if the index is out of range for the list of supported rinses.
      */
     virtual CHIP_ERROR GetSupportedRinseAtIndex(size_t index, NumberOfRinsesEnum & supportedRinse) = 0;
+
+    virtual Protocols::InteractionModel::Status PreAttributeCheck(AttributeId attributeId, uint16_t size, uint8_t * value) = 0;
```

Comment:
This needs to be documented.  As someone trying to implement this, I would have no idea what this is, what the arguments mean, or what I am supposed to do with it.  Especially given that this is required-to-implement.

But also, see comments above: should this be on the delegate at all?

Reply:
Will move them to the cluster handler